Changelog.md

```diff
@@ -1,5 +1,17 @@
 # v0.12.1 2024-05-11
 
+* [#1444](https://github.com/mbj/mutant/pull/1444)
+
+  Add ordering mutations to full and light operator set,
```

Comment:
I am slightly confused as to whether or not these were added to the full operator set because i think the deletion logic makes this just the full set and you also say:

>  People who do not care about ordering of their collections can now choose the light operator s

Reply:
@dgollahon Sorry for the late reply, the light operators do not include these operators. I've clarified the readme somewhat.